src/PageLayout/PageLayout.tsx

```diff
@@ -286,6 +286,7 @@ const Pane: React.FC<PageLayoutPaneProps> = ({
   const computedDividerWhenNarrow = dividerWhenNarrow === 'inherit' ? divider : dividerWhenNarrow
   return (
     <Box
+      as="aside"
```

Comment:
Waiting on an answer to this question: https://github.com/github/primer/issues/782#issuecomment-1069606557